openedx/features/partners/models.py

```diff
@@ -12,6 +16,7 @@ class Partner(TimeStampedModel):
     main_logo = models.CharField(max_length=255)
     small_logo = models.CharField(max_length=255)
     slug = models.CharField(max_length=100, unique=True)
+    configuration = JSONField(null=False, blank=True, default={"USER_LIMIT": ""})
```

Comment:
I see we're adding a field USER_LIMIT can we add a validation on save so that only digits are entered.

Reply:
achieved by overriding the model clean() method, and throwing a proper validation error from there

Reply:
Amazing! Resolve at will